nservicebus/security/encryption.md

```diff
@@ -96,12 +108,19 @@ This allows you to explicitly handled the encryption and decryption of each valu
 
 ## Message Encryption
 
-Message encryption leverages the pipeline to apply encryption to the whole message body.
+Message encryption encrypts the complete message body. It can even include the message headers if these would contain sensitive data.
```

Comment:
This makes it sounds like message encryption is built into NSB. I know in a few lines we say we don't offer but could we reword slightly? "Message encryption is the process of encrypting the entire message payload."

Should we mention this makes ServiceControl ... interesting?